blaze-server/src/main/scala/org/http4s/server/blaze/BlazeServerBuilder.scala

```diff
@@ -419,9 +424,12 @@ class BlazeServerBuilder[F[_]] private (
 
 object BlazeServerBuilder {
   def apply[F[_]](executionContext: ExecutionContext)(implicit F: Async[F]): BlazeServerBuilder[F] =
+    apply[F].withExecutionContext(executionContext)
```

Comment:
Once we have a reasonable default, there's no reason to make this overload special.  Maybe we should deprecate it.  Or better yet, scalafix it away.

Reply:
scalafixing it away seems reasonable, I only left it for compat right now. Will update in the next round :)